api/pkg/api/stack_group.go

```diff
@@ -2,26 +2,31 @@ package api
 
 import (
 	"github.com/chanzuckerberg/happy/api/pkg/cmd"
+	"github.com/chanzuckerberg/happy/api/pkg/request"
 	"github.com/chanzuckerberg/happy/api/pkg/response"
 	"github.com/chanzuckerberg/happy/shared/model"
 	"github.com/gofiber/fiber/v2"
 )
 
 type StackHandler struct {
-	stack cmd.Stack
+	stack cmd.StackIface
```

Comment:
same with Iface as part of the name. The type already shows its an interface, so I don't see the benefit for add "Iface" to it. Is there a particular reason for this naming choice?

Reply:
I wanted `Stack` reserved for the struct below. I'll rename the StackIface to StackManager